pkg/db/db.go

```diff
@@ -52,13 +55,18 @@ func Open(file string) (DB, error) {
 		return nil, err
 	}
 
-	i := &impl{db: db, log: util.NewLogger("db", "DB")}
-	i.log.Info("Created DB at path: %s\n", file)
+	filterLogger, err := util.ApplyFilter(*config.GetConfig(), "dbDB", util.NewLogger())
```

Comment:
instead of this please pass the config and the logger to  to `func Open(lgr log.Logger,  cfg config.Config,file string)`, 
Also use ComponenName
```suggestion
	lgr, err := util.ApplyFilter(cfg, ComponenName, lgr)
```